src/routes/logs.js

```diff
@@ -46,7 +44,7 @@ async function write(req, res) {
   let body = req.body;
 
   if (!body.message) {
-    return res.status(500).send('Missing log message');
+    return res.status(412).send('Missing log message');
```

Comment:
I don't think `400` is appropriate either, but the link you gave says `422` is appropriate. How about that?